index.php

```diff
@@ -121,13 +128,13 @@
 	#
 	exec("{$FROTZ_EXE_PATH} -i -Z 0 {$GLOBALS['frotz_data']['path']} <{$STREAM_PATH}/$session_id.f_in", $output);
 
-	
+
```

Comment:
Normally, whitespace changes should not be included in a PR, unless there's a specific reason. Not sure if you did this on purpose, and I'm not sure why those tabs are there in the first place, so this is fine. Just putting a note in, that you would not normally include these.